resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -8,6 +8,9 @@ jruby-puppet: {
     # used by the `puppetserver gem` command line tool.
     gem-home: /opt/puppetlabs/server/data/puppetserver/jruby-gems
 
+    environment-vars: {
+      "GEM_PATH": "/opt/puppetlabs/server/data/puppetserver/vendored-jruby-gems:/opt/puppetlabs/server/data/puppetserver/jruby-gems"
+    }
```

Comment:
This is just a temporary hack, my intent is to expose this as a first-class `gem-path` setting.